driver/SCHEMA_VERSION

```diff
@@ -1 +1 @@
-2.15.1
+2.16.1
```

Comment:
```suggestion
2.16.0
```
I see no reason for keeping the patch version number to 1

Reply:
🤦